docs/development/testguide.rst

```diff
@@ -227,38 +251,21 @@ Running tests in parallel
 -------------------------
 
 It is possible to speed up astropy's tests using the `pytest-xdist
-<https://pypi.org/project/pytest-xdist>`_ plugin.  This plugin can be
-installed using `pip`_::
+<https://pypi.org/project/pytest-xdist>`_ plugin.
 
-    pip install pytest-xdist
+Once installed, tests can be run in parallel using the ``'-n'``
+commandline option. For example, to use 4 processes::
 
-Once installed, tests can be run in parallel using the ``'--parallel'``
-commandline option.  For example, to use 4 processes::
+    pytest -n 4
 
-    python setup.py test --parallel=4
-
-Pass ``--parallel=auto`` to create the same number of processes as cores
+Pass ``-n auto`` to create the same number of processes as cores
 on your machine.
 
-Similarly, this feature can be invoked from Python::
+Similarly, this feature can be invoked from ``astropy.test``::
 
     >>> import astropy
     >>> astropy.test(parallel=4)
 
-Running tests to catch permissions errors
------------------------------------------
-
-It is possible to write code or tests that write into the source directory. This
-is not desirable because Python packages can be (and frequently are) installed
-in locations where the user may not have write permissions.  To check for these
-cases, the test runner has an option to have the test-runner
-directory be set as read-only to ensure the tests are not writing to that
-location.  This mode can be triggered by running the tests like so::
-
-    python setup.py test --readonly
```

Comment:
Is this automatically done by `tox`?

Reply:
No, this is the one thing that hasn't been translated to the new infrastructure yet. I'll have a think about it.

Reply:
OK, no worries - I'd just open a new issue about it, so we can go towards merging this!

Reply:
See #9852